SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -17,9 +18,18 @@ To see if there were any error during the data import, you can check the **Info*
 
 If the error is handled by the Power BI Connector, the pop-up will contain the error code and error message what you can find in [Errors raised by the Power BI Connector] section to learn more about the error.
 
+#### An error occurs when connecting via the Power BI Service.
+
+If an error occurs when a new connection is created, Power BI Service (Service) will show the exact error message.

```

Comment:
"the exact error message": do we mean "specific" error message here? 
Or does this mean the error message is shown as-is, not truncated, not condensed, etc.?

Reply:
Specific. :D